app/views/layouts/embedded.html.haml

```diff
@@ -5,6 +5,7 @@
     %meta{ name: 'robots', content: 'noindex' }/
 
     = stylesheet_pack_tag 'common', media: 'all'
+    = stylesheet_pack_tag current_theme, media: 'all'
```

Comment:
In most cases, embeds will be seen by non-logged in users. So I think it should be fixed to default theme, and it shouldn't be changed by user preferences.